sycl/test-e2e/DeprecatedFeatures/deprecated_sycl_device_filter.cpp

```diff
@@ -1,6 +1,6 @@
-// RUN: %{build} -o %t.out
-// RUN: env SYCL_DEVICE_FILTER='*' %{run-unfiltered-devices} %t.out &> %t.log
-// RUN: FileCheck %s < %t.log
+// RUN: %if !preview-breaking-changes-supported %{ %{build} -o %t.out %}
```

Comment:
I don't think we need any changes in the tests using old filter - it's the default still.